app/src/components/layout/AppBarBottom.vue

```diff
@@ -4,6 +4,17 @@
   >
     <div class="flex w-full justify-around max-w-96">
       <router-link
+        v-if="liteMode"
+        :to="{ name: 'search' }"
+        class="flex flex-col items-center w-1/3 p-2"
+        :class="{ 'text-blue-400': activeArea === 'map' }"
+        :aria-label="t('search')"
+      >
+        <i-ph-magnifying-glass-bold class="w-6 h-6 mb-1" />
+        <span class="mt-auto text-xs">{{ t('search').split(' ')[0] }}</span>
```

Comment:
@anbraten Maybe we should just put `...` in the template until there is a language that handles dots differently. :see_no_evil:

Reply:
@lukashass The idea is not to bad. `...` isn't / shouldn't be part of the translatable string somehow.